.changeset/tame-flies-confess.md

```diff
@@ -0,0 +1,14 @@
+---
+'astro': minor
+---
+
+build.format: 'preserve' to preserve source structure in final build
```

Comment:
```suggestion
Adds a new possible value to the `build.format` configuration: 'preserve'. It will preserve your source structure in the final build
```